pkg/proxy/backend/authenticator_test.go

```diff
@@ -235,9 +232,6 @@ func TestCustomAuth(t *testing.T) {
 		require.Equal(t, reUser, ts.mb.username)
 		require.Equal(t, reAttrs, ts.mb.attrs)
 		require.Equal(t, reCap&pnet.ClientDeprecateEOF, pnet.Capability(ts.mb.capability)&pnet.ClientDeprecateEOF)
-		host, _, err := net.SplitHostPort(inAddr)
-		require.NoError(t, err)
-		require.Equal(t, host, "::1")
```

Comment:
While I do want to keep the test, the logic of `ClientAddr()` moves to `BackendConnMgr` instead of `Authenticator`. Also, this test will fail on linux without IPV4. So I just removed it, maybe test `BackendConnMgr` more later.